### PR TITLE
Components

### PR DESCRIPTION
**Pull request by vanillainanutshell**
# What was done in this PR

- Add calculator.js component

### CODE REVIEW

**Review by er-contreras — COMMENTED** [review]
Hi @vanillainanutshell , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

wrong way

## Invalid Code Review Request

You have submitted a project, that has Linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+import Calculator from './components/Calculator';
```
Comment (er-contreras) [inline src/App.js:4]: - [ ] The path of your component does not match. Please fix it to be able to proceed with the project 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @vanillainanutshell, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (vanillainanutshell) [reply src/App.js:4]: Hi, what do you mean by "path does not match"?

Comment (er-contreras) [reply src/App.js:4]: You have a component called calculator.js and you are calling it Calculator.js that throws a path error.

Just change " import Calculator from './components/Calculator'; " for " import Calculator from './components/calculator'; "  in lower case

Comment (vanillainanutshell) [reply src/App.js:4]: image
 actually it shows w capped C in my local

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @vanillainanutshell, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**